uui/components/datePickers/__tests__/DatePicker.test.tsx

```diff
@@ -4,24 +4,23 @@ import {
 } from '@epam/uui-test-utils';
 import { DatePicker, DatePickerProps } from '../DatePicker';
 
-async function setupDatePicker(params: Partial<DatePickerProps>) {
+async function setupDatePicker(params: Pick<DatePickerProps, 'value' | 'format' | 'filter'>) {
```

Comment:
Let's also add tests for:
 - entering date from keyboard in target input. It will also support entering date not only in default format, but also in some others supported by DatePicker formats.
- isHoliday prop